src/test/ui/borrowck/borrowck-block-unint.stderr

```diff
@@ -7,6 +7,11 @@ LL |     force(|| {
    |           ^^ `x` used here but it isn't initialized
 LL |         println!("{}", x);
    |                        - borrow occurs due to use in closure
+   |
+help: use `=` to assign some value to `x`
+   |
+LL |     let x: isize = 0;
```

Comment:
nit: shouldn't this underline only the `= 0;` part?

Reply:
fixed!